### PR TITLE
Roles updates

### PR DESCRIPTION
**Pull request by lukefretwell**

### CODE REVIEW

File: _layouts/role.html [hunk 1]
```diff
@@ -9,21 +9,45 @@
...
+                <!--
                 {% assign sidenav = site.roles.navigation[page.sidenav] | default: page.sidenav %}
                 {% if sidenav %}
                 <div class="usa-layout-docs__sidenav desktop:grid-col-3{% if page.sticky_sidenav %} usa-sticky-sidenav{% endif %}">
                     {% include sidenav-roles.html links=sidenav %}
                 </div>
                 {% endif %}
+            -->
                 <div class="usa-layout-docs usa-layout-docs__main usa-prose{% if sidenav %} desktop:grid-col-9{% endif %}">
+                    <!--
```
Comment (dmundra) [inline _layouts/role.html:28]: Removing this is causing an accessibility error

```
Errors in http://localhost:4000/roles/executive:

 • Page must contain a level-one heading
   (https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one?application=axeAPI)

   (html)

   <html lang="en-US" class="fa-events-icons-ready"><head> <meta
   name="viewport...</html>
```

**Review by dmundra — CHANGES_REQUESTED** [review]
I see that you removed the header from roles.yml to use the hero feature. We just the hero feature added to the other roles.

File: _layouts/role.html (hunk 1 above)
Comment (lukefretwell) [reply _layouts/role.html:28]: @dmundra ah, I just did it for one page in the front matter and didn't think it would impact all of them. Should work now.

**Review by dmundra — APPROVED** [review]
Looking good @lukefretwell!

**Pull request merged**